client/state/data-layer/wpcom/read/teams/index.js

```diff
@@ -3,9 +3,20 @@
  */
 import { READER_TEAMS_REQUEST, READER_TEAMS_RECEIVE } from 'state/action-types';
 import wpcom from 'lib/wp';
+import inflight from 'lib/inflight';
 
 export function handleTeamsRequest( store, action, next ) {
-	wpcom.req.get( '/read/teams', { apiVersion: '1.2' } )
+	const dedupeKey = READER_TEAMS_REQUEST;
+	if ( inflight.requestInflight( dedupeKey ) ) {
```

Comment:
Did QueryComponents already do that?
I wouldn't want to introduce a new requirement